web/tests/projects/cpp/Makefile

```
@@ -1,29 +1,22 @@
-all:
-	$(CXX) -c call_and_message.cpp -Wno-all -Wno-extra
-	$(CXX) -c divide_zero.cpp -Wno-all -Wno-extra -Wno-division-by-zero
-	$(CXX) -c divide_zero_duplicate.cpp -Wno-all -Wno-extra
-	$(CXX) -c file_to_be_skipped.cpp -Wno-all -Wno-extra
+OBJS = $(SRCS:.cpp=.o)
+
+CXXFLAGS = -Wno-all -Wno-extra -Wno-division-by-zero
+
+SRCS = call_and_message.cpp divide_zero.cpp divide_zero_duplicate.cpp file_to_be_skipped.cpp new_delete.cpp null_dereference.cpp stack_address_escape.cpp skip_header.cpp path_begin1.cpp path_begin2.cpp statistical_checkers.cpp
```

It would be better to format this like so:

```suggestion
SRCS = call_and_message.cpp \
       divide_zero.cpp \
       divide_zero_duplicate.cpp \
       file_to_be_skipped.cpp \
       new_delete.cpp \
       null_dereference.cpp \
       stack_address_escape.cpp \
       skip_header.cpp \
       path_begin1.cpp \
       path_begin2.cpp \
       statistical_checkers.cpp
```
It would be more readable.